client/src/components/App.jsx

```diff
@@ -1,29 +1,63 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
-import Review from './Review';
+import Review from './review';
 import Search from './Search';
 import Ratings from './Ratings';
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
-    allReviews: [],
+      allReviews: [],
+      currentReviews: [],
+      currentSearchTerm: null,
     }
+    this.searchSubmit = this.searchSubmit.bind(this);
+    this.searchBarTextChange = this.searchBarTextChange.bind(this);
   }
+  
 
   componentDidMount() {
     const component = this; 
     axios('http://localhost:3000/locations/1/reviews')
-    .then(location => component.setState({allReviews: location.data.reviews}));
+    .then(location => {
+      component.setState(
+        {allReviews: location.data.reviews,
+         currentReviews: location.data.reviews,
+        }
+      )
```

Comment:
Missing semicolon:
28 (Closing paren to component.setState)
35 (Closing paren to this.setState)
51 (Closing paren to this.setState)
64 (Closing paren to expression in return statement)